src/0.9.1/quickstart.md

```diff
@@ -0,0 +1,182 @@
+# Quickstart
+
+To get started extremely quick using Docker:
+
+```bash
+docker network create immudbnet
+docker run -d --net immudbnet -it --rm --name immudb -p 3322:3322 codenotary/immudb:latest
+docker run -it --rm --net immudbnet --name immuclient codenotary/immuclient:latest -a immudb
+```
+
+## Download the binaries
+
+### General
+
+[Get the latest builds](https://github.com/codenotary/immudb/releases/latest)
+
+### macOS specific
+
+The community already added immudb to [HomeBrew](https://formulae.brew.sh/formula/immudb), therefore you can simply run
+```bash
+brew install immudb
+```
+
+In case you want to run immudb as a service, please check the following [guideline](https://medium.com/swlh/how-to-use-launchd-to-run-services-in-macos-b972ed1e352).
+
+
+## Build the binaries
+
+To build the binaries yourself, simply clone this repo and run
+
+```bash
+make all
+```
+
+
+### Linux (by component)
+
+```bash
+GOOS=linux GOARCH=amd64 make immuclient-static immuadmin-static immudb-static
+```
+
+### MacOS (by component)
+
+```bash
+GOOS=darwin GOARCH=amd64 make immuclient-static immuadmin-static immudb-static
+```
+
+### Windows (by component)
+
+```bash
+GOOS=windows GOARCH=amd64 make immuclient-static immuadmin-static immudb-static
+```
+
+## First start
+
+### Run immudb binary
```

Comment:
Previously I was referring to this section that would be nice to specify also how to stop it, especially when the user might want to start it in the background.